### PR TITLE
[R4R] sig library integration

### PR DESCRIPTION
**Pull request by denalimarsh**
This branch has the kava-labs/sig dependency and relevant updates.

### CODE REVIEW

File: example/cdp.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+const kavaUtils = require("../src/utils").utils;
+const KavaClient = require("../src/client").KavaClient;
+
+const KAVA_API_TESTNET_6000_INTERNAL = "http://54.196.2.124:1317";
+const kavaAddress = "kava1g0qywkx6mt5jmvefv6hs7c7h333qas5ks63a6t";
+const kavaMnemonic =
+  "lecture draw addict sea prefer erupt army someone album liquid sadness manual fence vintage obey shrimp figure retreat kick refuse verify alien east brand";
+
+const BNB_CONVERSION_FACTOR = 10 ** 8;
```
Comment (karzak) [inline example/cdp.js:11]: let's create an issue to add a client method for `getConversionFactor`, I've noticed that I'm either hard coding it or reusing a lot of code to get it - would be nicer to have on the client.

File: example/cdp.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const _ = require("lodash");
```
Comment (karzak) [inline example/cdp.js:1]: add `.npmignore` file with 

```
node_modules
example
```

to avoid publishing examples

File: example/cdp.js (hunk 1 above)
Comment (denalimarsh) [reply example/cdp.js:11]: https://github.com/Kava-Labs/javascript-sdk/issues/21

File: example/cdp.js (hunk 2 above)
Comment (denalimarsh) [reply example/cdp.js:1]: https://github.com/Kava-Labs/javascript-sdk/pull/20/commits/c64635c825e8c08f6ff972d7a36abe852a75c2c4

**Pull request merged**